Navidux/Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Navidux",
+    platforms: [.iOS(.v13)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
```

Comment:
There is no need in default comments in `Package.swift`

Reply:
remove comments